README.md

```diff
@@ -4,8 +4,9 @@ This an extension for the [MagicMirror](https://github.com/MichMich/MagicMirror)
 ## Installation
 1. Navigate into your MagicMirror's `modules` folder and execute `git clone https://github.com/paviro/MMM-PIR-Sensor.git`. A new folder will appear navigate into it.
 2. Execute `npm install` to install the node dependencies.
-3. Add your user (`pi`?) to the `gpio group` by executing `sudo useradd -g pi gpio`.
-4. Reboot your Pi.
+3. Add your user (`pi`?) to the `gpio group` by executing `sudo usermod -a -G gpio pi`.
+4. Execute `sudo chmod u+s /opt/vc/bin/tvservice && sudo chmod u+s /bin/chvt` to allow turning on/off the hdmi output.
```

Comment:
Is changing the permission of a system file really a better idea than using sudo (which sucks too I know)?